### ISSUE TITLE
How to load the annotation file

### ISSUE THREAD

**Issue by E-LEAF** [RockyXu66/Faster_RCNN_for_Open_Images_Dataset_Keras#13]
Thanks for the GitHub repository and easy use in colab. I want to train with my custom dataset. I have the annotations in txt format. I tried changing the file path in 'base_path' and 'train_path' to my path in the training notebook. But I got the following error. Maybe I assigned the path wrongly. Can you help me. 
![frcnn](https://user-images.githubusercontent.com/40974215/54863862-111a6b80-4d74-11e9-8ae0-86cf274c62a6.JPG)

**Comment by RockyXu66** [comment]
According to your code. I think you mistype the **train_path** which should be a .txt file. You can see the code you comment.
![image](https://user-images.githubusercontent.com/18605943/55060164-ccf8d500-50ab-11e9-863c-92924c23a087.png)

**Comment by E-LEAF** [comment]
I didn't understand  **train_path** - drive/My Drive/AI/ Dataset/ Open Image Dataset v4(Bounding Boxes)/person_car_phone_train_annotation.txt
Did you combine all the annotation files into a single file?
I have my annotation files in XML format and each image has an individual annotation XML file. I just put them into a file and assigned a path to it. Did I miss something?
One more classification - does base_path refer to the file containing training images?

**Comment by DTW1004** [comment]
hello sir, I meet a problem, could you please tell me  how to get the four file for training?
![image](https://user-images.githubusercontent.com/43997072/56503762-06a4ea80-6549-11e9-800a-ae77521a023a.png)
the 'model_frcnn_vgg.hdf5','record.csv' and 'model_vgg_config.pickle' where to get?? if you know can you give me the link ? thank you

**Comment by devendraswamy** [comment]
base_path = " this is the path  which you have the total file (supposed to train and tasting )

train_path = " here you have to mentioned your annotations.txt file path with annotation.txt.     like ==/home/dell/Documents/DEVENDRA/FasterRCNN/FRCNN_opendata/test_annotation.txt'

1
output_weight_path = os.path.join(base_path, 'model/model_frcnn_vgg.hdf5')

record_path = os.path.join(base_path, 'model/record.csv') # Record data (used to save the losses, classification accuracy and mean average precision)

base_weight_path = os.path.join(base_path, 'model/vgg16_weights_tf_dim_ordering_tf_kernels.h5')

config_output_filename = os.path.join(base_path, 'model_vgg_config.pickle')       2
 

for running 1-2 path you want to create just " model " folder in that base_path

**Comment by mehnaz1985** [comment]
@RockyXu66 , if I want to use pascal_voc dataset here, how could I incorporate that? do u have any link that I can directly use in the training phase?
thanks in advance

**Comment by devendraswamy** [comment]
U have to download that from official website or from any open source.

https://github.com/rbgirshick/py-faster-rcnn#beyond-the-demo-installation-for-training-and-testing-models

In above u can find the dataset. Please read it carefully. Thank u.

**Comment by mehnaz1985** [comment]
@devendraswamy thanks. I can download that from the link but how could I convert it in faster rcnn format? i.e. I would like to convert it into annotation.txt and test_annotation.txt format. only then I could use that in this code, right? in the pascal_voc, there are individual folders.

**Comment by devendraswamy** [comment]
Just change the directory locations in code and run training script.Thank u

**Comment by devendraswamy** [comment]
Above one is For tesorflow faster rcnn only other wise u will write a code
for extracting required data from the xml files and then save that data
into txt file for this repo training

**Comment by mehnaz1985** [comment]
Hmmm, that's the problem!  have to convert from XML to txt. btw thanks for
your time and suggestions.


Mehnaz Tabassum
PhD Candidate
School of SEIT
Federation University, Gippsland Campus
Victoria, Australia.

**Comment by mehnaz1985** [comment]
@RockyXu66 @devendraswamy 
why no of test image is larger than train image? 
![image](https://user-images.githubusercontent.com/66414863/113411438-77c1cd80-9401-11eb-801c-994262ce5684.png)

**Comment by mehnaz1985** [comment]
![image](https://user-images.githubusercontent.com/66414863/113411479-9758f600-9401-11eb-851d-cbd3fef946e6.png)

**Comment by devendraswamy** [comment]
Did you tried data preprocessing jupyter file in this repo , that should be
help you to create a data for train and test.

**Comment by Ziyang0612** [comment]
@devendraswamy 

> base_path = " this is the path which you have the total file (supposed to train and tasting )

train_path = " here you have to mentioned your annotations.txt file path with annotation.txt. like ==/home/dell/Documents/DEVENDRA/FasterRCNN/FRCNN_opendata/test_annotation.txt'

1
output_weight_path = os.path.join(base_path, 'model/model_frcnn_vgg.hdf5')

record_path = os.path.join(base_path, 'model/record.csv') # Record data (used to save the losses, classification accuracy and mean average precision)

base_weight_path = os.path.join(base_path, 'model/vgg16_weights_tf_dim_ordering_tf_kernels.h5')

config_output_filename = os.path.join(base_path, 'model_vgg_config.pickle') 2

for running 1-2 path you want to create just " model " folder in that base_path


As you mentioned at here, I just need to create " model " folder in that base_path for running 1-2 path. But i still cannot get the file. Is there any solution?

**Comment by Ziyang0612** [comment]
I only can get the model_vgg_config.pickle file but the csv and h5 file don't have
